<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNewArcGisRestConnectionBase</class>
 <widget class="QDialog" name="QgsNewArcGisRestConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New Connection</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="0">
    <widget class="QgsScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>628</width>
        <height>741</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="mGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Connection Details</string>
         </property>
         <layout class="QGridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="TextLabel1">
            <property name="text">
             <string>URL</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txtUrl">
            <property name="toolTip">
             <string>HTTP address of the Web Map Server</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txtName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Name of the new connection</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TextLabel1_2">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="TextLabel1_5">
            <property name="text">
             <string>Prefix</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="mUrlPrefix">
            <property name="toolTip">
             <string>HTTP address Proxy Prefix of the Web Map Server</string>
            </property>
            <property name="placeholderText">
             <string>https://mysite.com/proxy.jsp?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>ArcGIS Portal Details</string>
         </property>
         <layout class="QGridLayout" name="gridLayout1">
          <item row="0" column="1">
           <widget class="QLineEdit" name="mCommunityEndPointLineEdit">
            <property name="toolTip">
             <string>HTTP address of the Web Map Server</string>
            </property>
            <property name="placeholderText">
             <string>https://mysite.com/portal/sharing/rest/community/</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel1_3">
            <property name="text">
             <string>Community endpoint URL</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel1_4">
            <property name="text">
             <string>Content endpoint URL</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="mContentEndPointLineEdit">
            <property name="toolTip">
             <string>HTTP address of the Web Map Server</string>
            </property>
            <property name="placeholderText">
             <string>https://mysite.com/portal/sharing/rest/content/</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mAuthGroupBox">
         <property name="title">
          <string>Authentication</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QgsAuthSettingsWidget" name="mAuthSettings" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsHttpHeaderWidget" name="mHttpHeaders" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsHttpHeaderWidget</class>
   <extends>QWidget</extends>
   <header>qgshttpheaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtUrl</tabstop>
  <tabstop>mUrlPrefix</tabstop>
  <tabstop>mCommunityEndPointLineEdit</tabstop>
  <tabstop>mContentEndPointLineEdit</tabstop>
  <tabstop>mAuthSettings</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsNewArcGisRestConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsNewArcGisRestConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
